jinja.py

```diff
@@ -17,13 +17,13 @@
 UBER_PASS = os.environ.get('UBER_PASS') or ""
 
 JOBS = {}
-HOST = '172.23.98.63'
+HOST = '127.0.0.1'
 
 if len(sys.argv) == 2:
     HOST = sys.argv[1]
 
 
-def getJS(url, params=None, retry=5, append_api_json=True):
+def getJS(url, params=None, retry=10, append_api_json=True):
```

Comment:
Why raise it to 10?

Reply:
Ah yes, also local testing. Had a lot of retries so bumped it. Will put it back